lisp/ess-custom.el

```diff
@@ -646,211 +646,345 @@ regardless of where in the line point is when the TAB command is used."
 nil means to use R/S indentation.")
 (make-variable-buffer-local 'ess-indent-line-function)
 
-(defvar ess-indent-level 2
+(defvar ess-indent-level 4
   "Indentation of S statements with respect to containing block.")
 
-(defvar ess-brace-imaginary-offset 0
-  "Imagined indentation of an open brace following a statement.")
+(defvar ess-offset-statement '(t)
```

Comment:
I don't like this statement vs expression terminology. It comes from C and doesn't make sense in R. Let's stick to blocks. Like `ess-offset-block`. Is it what you meant here?